tests/dummy/app/templates/new-platform-flexberry-g-i-s-layer-metadata-e.hbs

```diff
@@ -13,6 +13,7 @@
 {{/if}}
 
 <form class="ui form flexberry-vertical-form" role="form">
+  {{flexberry-error error=error}}
```

Comment:
This template is unnecessary, because whole `new-platform-flexberry-g-i-s-layer-metadata-e` form is placed inside addon, it's template is placed in app/templates, so this template is obsolette and must be removed.